client/src/app/services/event/event.service.ts

```diff
@@ -45,4 +45,18 @@ export class EventService {
       })
       .pipe(switchMap(() => this.queryEventsPaginated(paginated)));
   }
+
+
+
+  join(id): Observable<Object> {
+    return this.httpClient.get(`http://localhost:3000/api/event/join/${id}`);
```

Comment:
This can be typed no?
this.httpClient.get<IEventSchema>(...)

the same applies to leave()